compiler/rustc_middle/src/ty/flags.rs

```diff
@@ -300,8 +301,12 @@ impl FlagComputation {
     }
 
     fn add_const(&mut self, c: ty::Const<'_>) {
-        self.add_ty(c.ty());
-        match c.kind() {
+        self.add_flags(c.flags());
+        self.add_exclusive_binder(c.outer_exclusive_binder());
+    }
+
+    fn add_const_kind(&mut self, c: &ty::ConstKind<'_>) {
```

Comment:
I actually am unsure if it's more efficient just to take the `ConstKind` by value here.